### DISCUSSION TITLE
Pimcore X migration problem: "There is no table with name 'translations_website' in the schema"

### DISCUSSION THREAD

**Discussion by akno77** [pimcore/pimcore#9160] [Q&A / Support] [answered]
Hi,

I'm currently trying to upgrade a Pimcore project from 6.8 to X. I first upgraded from 6.8 to 6.9 with no problems. Now I'm trying to upgrade von 6.9 to X.

Currently I get the following error during the composer update migration process:

`[notice] Migrating up to Pimcore\Bundle\CoreBundle\Migrations\Version20210505093841
[error] Migration Pimcore\Bundle\CoreBundle\Migrations\Version20210324152822 failed during Execution. Error: "There is no table with name 'website_dev.translations_website' in the schema."`

When I look into the DB I see a VIEW named "translations_website". I did backups of the 6.8 and 6.9 DBs. In the 6.8 DB "translations_website" is a TABLE, but with the 6.9 upgrade "translations_website" became a VIEW!

So of course the migration SQL statement on the "translations_website" VIEW fails:
`ALTER TABLE `'.$translationsTable.'` ADD COLUMN `type` varchar(10) DEFAULT NULL AFTER `key``

Is it correct, that "translations_website" is a VIEW in Pimcore X? If so, why is the migration script not functional? Everyone should run into this problem I'd expect...

Anyone more with this problem? How can I solve it?

**Comment by jdreesen** [comment, marked as answer]
Sounds like #9110 to me... will be fixed with `v10.0.1` then.

**Reply by akno77** [reply]
Thanks, yes this is the same problem. Didn't see this issue...sorry!
